_events/2015-08-29.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Music Games Hackathon"
+description: "Music Games Hackathon"
```

Comment:
Typo: Talks, hacks, and workshops focusing on the intersection of music and games'